13/memory-user.c

```diff
@@ -3,26 +3,33 @@
 #include <time.h>
 #include <unistd.h>
 
-int main(int argc, char *argv[]) {
-    if (argc != 3) {
+int main(int argc, char *argv[])
+{
+    if (argc != 3)
+    {
         fprintf(stderr, "usage: memory-user <memory> <time>\n");
         exit(EXIT_FAILURE);
     }
 
     printf("pid: %d\n", getpid());
 
-    int memory  = atoi(argv[1]) * 1000000;
+    int memory = atoi(argv[1]) * 1024 * 1024;
     int length = (int)(memory / sizeof(int));
     int runTime = atoi(argv[2]);
     int *arr = malloc(memory);
     clock_t begin = clock();
     double time_spent;
 
-    for (int i = 0; i < length; i++) {
+    for (int i = 0; i < length; i++)
+    {
+        arr[i] += 1;
+    }
+
+    while (1)
```

Comment:
Move `malloc`, `for` and `free`  inside `while` is more reasonable, IMHO.

Reply:
Awesome i agree